service/controller/resource/servicemonitor/create.go

```diff
@@ -0,0 +1,26 @@
+package servicemonitor
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	services, err := toServiceMonitors(obj)
```

Comment:
i reckon this should be `serviceMonitors` - `services` is confusing

Reply:
fixed